From: KevinDuan <duankaiwen@kylinos.cn>
Date: Fri, 14 Aug 2026 09:27:11 +0000
Subject: Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: openkylin-huanghe/ukui-panel-search
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/ukui-panel-search/it/
---
 widgets/ukui-panel-search/translations/org.ukui.panel.search_it.ts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/widgets/ukui-panel-search/translations/org.ukui.panel.search_it.ts b/widgets/ukui-panel-search/translations/org.ukui.panel.search_it.ts
index 194f565..afcec3d 100644
--- a/widgets/ukui-panel-search/translations/org.ukui.panel.search_it.ts
+++ b/widgets/ukui-panel-search/translations/org.ukui.panel.search_it.ts
@@ -5,11 +5,11 @@
     <name>main</name>
     <message>
         <source>Search</source>
-        <translation type="unfinished"></translation>
+        <translation>Cerca</translation>
     </message>
     <message>
         <source>Search settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Imposta elementi di ricerca</translation>
     </message>
 </context>
 </TS>
